zebra-network/Cargo.toml

```
@@ -10,7 +10,7 @@ edition = "2018"
 [dependencies]
 bitflags = "1.2"
 byteorder = "1.4"
-bytes = "0.6"
+bytes = "1.1"
```

Let's use full versions, to avoid duplicate dependencies.

```suggestion
bytes = "1.1.0"
```

---

Done.